lib/Runtime/Debug/DebugContext.cpp

```diff
@@ -141,6 +142,23 @@ namespace Js
 
     HRESULT DebugContext::RundownSourcesAndReparse(bool shouldPerformSourceRundown, bool shouldReparseFunctions)
     {
+        struct AutoRestore
+        {
+            AutoRestore(DebugContext* debugContext)
+                :debugContext(debugContext)
+            {
+                this->debugContext->isReparsingSource = true;
```

Comment:
> this->debugContext->isReparsingSource = true; [](start = 15, length = 46)

We should restrict this to only when shouldReparseFunctions is true. Not needed during PerformSourceRundown